clients/nft_bridge/main.ts

```diff
@@ -110,6 +110,48 @@ yargs(hideBin(process.argv))
 
         console.log(vm)
     })
+    .command('generate_upgrade_chain_vaa [chain_id] [contract_address]', 'create a VAA to upgrade a chain (debug-only)', (yargs) => {
+        return yargs
+            .positional('chain_id', {
+                describe: 'chain id to upgrade',
+                type: "number",
+                required: true
+            })
+            .positional('contract_address', {
+                describe: 'contract to upgrade to',
+                type: "string",
+                required: true
+            })
+            .option('guardian_secret', {
+                describe: 'Guardian\'s secret key',
+                type: "string",
+                default: "cfb12303a19cde580bb4dd771639b0d26bc68353645571a8cff516ab2ee113a0"
```

Comment:
Assuming this is for the tests or for tilt and not an actual real secret, right?

Reply:
Yes, this is the tilt / devnet key. I copied it from the token_bridge client, and it appears many places in the repo.